spec/requests/internal/users_spec.rb

```diff
@@ -1,8 +1,8 @@
 require "rails_helper"
 
 RSpec.describe "Internal::Users", type: :request do
-  let(:user) { create(:user, twitter_username: nil) }
-  let(:user2) { create(:user, twitter_username: "Twitter") }
+  let!(:user) { create(:user, twitter_username: nil) }
```

Comment:
here we make sure the users are created before the tests run, to make the test more robust